monai/deploy/operators/monai_bundle_inference_operator.py

```diff
@@ -363,6 +407,10 @@ def __init__(
             if self._bundle_path and self._bundle_path.is_file():
                 self._init_config(self._bundle_config_names.config_names)
                 self._init_completed = True
+            elif self._bundle_path and self._bundle_path.is_dir():
```

Comment:
Once the is_dir() case separated out, there would be no change needed here, I suppose.